src/Migration/Transfer.php

```
@@ -322,13 +358,17 @@ public function getReport(string $statusLevel = ''): array
     public static function extractServices(array $services): array
     {
         $resources = [];
+        $groupDatabasesIndex = array_search(Transfer::GROUP_DATABASES, $services);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$groupDatabasesIndex'. (undefined)

(UnusedLocalVariable)